### PR TITLE
Com 393 -Added columns for emergency contact info

### PR DESCRIPTION
**Pull request by rubailly**

### CODE REVIEW

File: ui/app/appointments/views/manage/list/listView.html [hunk 1]
```diff
@@ -35,13 +35,16 @@
                         </li>
                     </ul>
                 </td>
+                <td>{{::appointment.additionalInfo["Emergency contact name"]}}</td>
```
Comment (lauravignoli) [inline ui/app/appointments/views/manage/list/listView.html]: @rubailly The column titles don't align with the column values, please fix:

appointmentListColumns

**Comment by lauravignoli** [comment]
@rubailly Great solution, well done!!

**Comment by lauravignoli** [comment]
@rubailly Some columns are missing and the emergency details should be displayed after additional information as per requirement:

3. Add columns for **Patient Phone number**, **Village**, Emergency contact name, number, relationship after the Additional information

File: ui/app/appointments/views/manage/list/listView.html (hunk 1 above)
Comment (rubailly) [reply ui/app/appointments/views/manage/list/listView.html]: You might have missed the CSS when compiling the code. [This commit ](https://github.com/jembi/cameroon-openmrs-module-bahmniapps/pull/99/commits/c3de0fcb33312736be69fb40e444bd939e36572b) is actually the one adjusting the column titles with their corresponding values.
Screenshot from 2019-07-01 11-35-07

**Comment by rubailly** [comment]
I totally missed the two columns -sorry. I'm adding those now and moving them after Additional information column.

File: ui/test/unit/appointments/controllers/manage/list/appointmentsListViewController.spec.js [hunk 2]
```diff
@@ -714,6 +714,11 @@ describe('AppointmentsListViewController', function () {
                 {heading: 'APPOINTMENT_WALK_IN', sortInfo: 'appointmentKind', enable: true},
                 {heading: 'APPOINTMENT_SERVICE_LOCATION_KEY', sortInfo: 'location.name', class: true, enable: true},
                 {heading: 'APPOINTMENT_ADDITIONAL_INFO', sortInfo: 'additionalInfo', class: true, enable: true},
```
Comment (lauravignoli) [inline ui/test/unit/appointments/controllers/manage/list/appointmentsListViewController.spec.js]: @rubailly Please remove this heading.

**Pull request merged**